appveyor.yml

```diff
@@ -0,0 +1,53 @@
+build: false
+shallow_clone: true
+platform: x86
+clone_folder: c:\projects\symfony
+
+environment:
+    matrix:
+        - PHP_EXT: 1
+        - PHP_EXT: 0
+
+cache:
+    - c:\php -> appveyor.yml
+    - .phpunit -> phpunit
+    - vendor -> composer.json
+
+init:
+    - SET PATH=c:\php;%PATH%
+    - SET COMPOSER_NO_INTERACTION=1
+    - SET SYMFONY_DEPRECATIONS_HELPER=weak
+    - SET PHP=1
+
+install:
+    - IF EXIST c:\php (SET PHP=0) ELSE (mkdir c:\php)
+    - cd c:\php
+    - IF %PHP%==1 appveyor DownloadFile http://windows.php.net/downloads/releases/archives/php-5.4.43-nts-Win32-VC9-x86.zip
```

Comment:
@nicolas-grekas why testing against PHP 5.4 which is nearly EOL rather than against PHP 5.6 ?